### DISCUSSION TITLE
Open-source the Markdoc website?

### DISCUSSION THREAD

**Discussion by sambarrowclough** [markdoc/markdoc#8] [Q&A] [answered]
Just checked out the markdoc docs website, and it's incredible. Is there any chance you will be open-sourcing the site itself?

**Comment by mfix-stripe** [comment, marked as answer]
> Just checked out the markdoc docs website, and it's incredible. Is there any chance you will be open-sourcing the site itself?

@eeekkoo done! https://github.com/markdoc/docs

**Reply by sambarrowclough** [reply]
legend!

**Reply by tdrhq** [reply]
This is awesome! @mfix-stripe How do you use the live-edit functionality internally? Ideally I'd like to use the live edit on every page to be able to modify the content of the page while I'm looking at it, possibly only in dev mode.

**Reply by mfix-stripe** [reply]
The live-edit feature is mostly a demonstration of how Markdoc works, but it also let's us tweak content live and see how it would look in-situ, instantly. You can open the live-edit panel on any page with <kbd>CMD+J</kbd> 🙂 

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/62121649/168339277-29e94af2-03f0-4b78-ab5f-a557727899c9.png">

**Reply by tdrhq** [reply]
Ah interesting! I didn't realize that's what the "CMD+J" comment in the editor panel meant.

So do technical writers at Stripe typically just use the editor of their choice to edit pages? And the pages are stored in a repo as opposed to some sort of CMS, right?

**Reply by mfix-stripe** [reply]
@tdrhq that's how it currently works, yes 🙂

**Reply by tdrhq** [reply]
Good to know! Thanks so much for all of the insight!

**Reply by samuelstroschein** [reply]
@mfix-stripe The live editing is interesting. Can contributors contribute directly from the preview?

**Reply by mfix-stripe** [reply]
Not at the moment, no! You can just edit the source code directly on GitHub, though: https://github.com/markdoc/docs

**Comment by radsmine** [comment]
@mfix-stripe Hey, all this while, questions regarding the tools that Stripe was using to create such comprehensive , detailed documentation, have been answered! Thanks for sharing the main ingredient of this fantastic recipe! 

Can you throw some light on:
_ How is that content reviewed? Any tools that facilitate reviews across different teams
_ Where is this content stored? GitHub repo ?
_ How is this content published? using GitHub actions?

**Reply by mfix-stripe** [reply]
> How is that content reviewed? Any tools that facilitate reviews across different teams
Just with GitHub PRs! Here are a bunch of examples: https://github.com/markdoc/docs/pulls?q=is%3Apr+is%3Aclosed

> Where is this content stored? GitHub repo ?
https://github.com/markdoc/docs

> How is this content published? using GitHub actions?
We just deploy the site using Vercel. Right now it is manual, but it will autodeploy from GitHub in the future.
